src/tests/Interop/IJW/getRefPackFolderFromArtifacts.ps1

```diff
@@ -0,0 +1,13 @@
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+$interopFolder = Split-Path $PSScriptRoot -Parent
+$testsFolder = Split-Path $interopFolder -Parent
+$srcFolder = Split-Path $testsFolder -Parent
+$repoRoot = Split-Path $srcFolder -Parent
+
+$versionPropsFile = "$repoRoot/eng/Versions.props"
+
+$majorVersion = Select-Xml -Path $versionPropsFile -XPath "/Project/PropertyGroup/MajorVersion" | %{$_.Node.InnerText}
+$minorVersion = Select-Xml -Path $versionPropsFile -XPath "/Project/PropertyGroup/MinorVersion" | %{$_.Node.InnerText}
+
+echo "$repoRoot/artifacts/bin/ref/net$majorVersion.$minorVersion"
```

Comment:
Thoughts on writing out to stdout if this path doesn't exist? I am trying to ensure we know what failed if the CMake process fails. Same for the `getRefPackFolderFromSdk.ps1`